test/amp-caching/amp-caching-test.js

```diff
@@ -0,0 +1,79 @@
+describe('Amp caching', function() {
+  let driver;
+
+  before(async () => {
+    driver = global.__AMPSW.driver;
```

Comment:
Is there a reason `driver` couldn't just be defined when its created?

`const driver - global.__AMPSW.driver;`

Reply:
fixed